public/components/GeoPlot.vue

```diff
@@ -755,11 +754,11 @@ export default Vue.extend({
         }).$mount();
 
         // Add interactivity to the layer.
-        layer.bindTooltip(tooltip.$el as HTMLElement).on("click", () => {
-          if (this.isRemoteSensing) {
+        if (this.isRemoteSensing) {
```

Comment:
Sorry, but I think we still want to see the tooltip, just not have the `.on("click", ...` event listener.
More like:
```
layer.bindTooltip(tooltip.$el as HTMLElement);
if (this.isRemoteSensing) {
    layer.on("click", () => { this.showImageDrilldown(imageUrl, item) });
}
```